packages/badge-service/scoreProcessor/scoreProcessor.ts

```diff
@@ -0,0 +1,72 @@
+const sha512 = require('js-sha512');
+import { MutationScoreMapper, ProjectMapper, MutationScore } from 'stryker-dashboard-data-access';
+
+export async function run(context: any, req: any) {
+    let statusCode = 400;
+    let responseBody;
+
+    context.res = {
+        status: statusCode,
+        body: responseBody
+    };
+
+    if (typeof req.body == 'object') {
+        if(!req.body.apiKey || !req.body.repositorySlug || !req.body.mutationScore)
+        {
+            return;
+        }
+        const hash = sha512.sha512_256(req.body.apiKey);
```

Comment:
We should use the same hashing algorithm in the website. Let's move this to a shared utility function later on.